src/parsec_service/stability.md

```diff
@@ -0,0 +1,68 @@
+# Parsec Stability
+
+Work in progress, see the [related issue](https://github.com/parallaxsecond/parsec-book/issues/83).
+
+## Why is stability important?
+
+Parsec is a long-running service that will be configured, deployed, upgraded and maintained over a
+long period of time. The environment will also have client applications that are calling it, and
+persistent state that is collected on storage. Components will be upgraded over time. For Parsec to
+be supportable in enterprise system, it must not be brittle against upgrades. If Parsec is upgraded
+and restarted, client applications should continue working, and all persistent state from prior
+versions should remain valid and usable.
+
+## What does stability mean for the Parsec service?
+
+The following definition of stability is chosen for the Parsec service: *whatever two versions A and
+B of the Parsec binary, B being newer than A, A and B are stable if A can be removed and replaced by
+B without breaking anything*.
+
+Note that we are not looking at stability in the reverse direction: B can not be replaced by A
+without breaking anything.
+
+The principle of [semantic versioning](https://semver.org/) is used to describe the stability of
+Parsec versions and when breaking changes are done. If `parsec` version is at `1.0.0` then all
+future stable version to that will be `1.x.y`.
```

Comment:
I agree that other than describing stability, being stable for a long time is the main goal here. With that paragraph I did not want to imply that using semver would be a reason for us to submit breaking changes often but more as a way to describe our stability. Even if it never happens (what is our goal), the semver definition allows us to prepare for that eventuality.
Moreover I would say that it would be a good sign for stakeholders to see that Parsec is still at `1.x.y` after a very long time.

I will add a clarification sentence that staying stable in the same `1.x.y` is indeed the goal here.